microcosm_pubsub/dispatcher.py

```diff
@@ -61,6 +61,7 @@ def __init__(self, graph):
         self.send_metrics = graph.pubsub_send_metrics
         self.send_batch_metrics = graph.pubsub_send_batch_metrics
         self.max_processing_attempts = graph.config.sqs_message_dispatcher.message_max_processing_attempts
+        self.sentry_enabled = graph.sentry_logging.enabled
```

Comment:
Should this be config on the dispatcher, or more generic like this? Would we push sentry logs outside of this?

Reply:
Potentially, sentry is moving in a direction of being more than just error reporting, so you push any log like data that is relevant and tag it accordingly so you can quickly find traces/related data.

Also my thinking was keeping the logic for deciding if sentry is enabled quite generic so that if this goes well. The logic for checking if sentry is enabled and adding it to the graph and the cleaning functions moves to it's own library that would be used by both pubsub/flask libs.